packages/client/src/views/SysAdmin/Team/user/userCreation/CreateNewUser.tsx

```diff
@@ -82,6 +83,11 @@ const Container = styled.div`
 class CreateNewUserComponent extends React.Component<Props & IDispatchProps> {
   async componentDidMount() {
     const { userId, client } = this.props
+    if (
+      this.props.match.path.includes(CREATE_USER_ON_LOCATION.split('/:')[0])
+    ) {
+      this.props.clearUserFormData()
```

Comment:
Would it be possible to do this in the `componentWillUnmount` hook of this component? So every time the component unmounts. I've noticed matching the URL path can sometimes be a bit fragile